src/grid/tests.rs

```diff
@@ -93,7 +94,7 @@ fn test_iter() {
         }
     }
 
-    info!("grid: {:?}", grid);
+    println!("Grid: {:?}", grid);
```

Comment:
I thought I removed all printlns in the code when changing the logging, but it looks like I forgot some. All printlns in test code can be removed, I suspect they were mainly for debugging failures.